man/nheko.1.adoc.in

```diff
@@ -147,6 +172,15 @@ Inserts `┯━┯╭( º _ º╭)`
 */sovietflip*::
 Inserts `ノ┬─┬ノ ︵ ( \\o°o)\\`
 
+=== Inline commands
+
+*:*::
+Open unicode emoji picker.
+
+*~*::
```

Comment:
Maybe you should mention @, # and <tab> for completeness sake then too, if you name the completers "inline commands" ;p

Reply:
Haha, yeah. I totally forgot. I'm open to suggestions for a better term, btw.. Shold I call them completers instead?

Reply:
Yeah, completers is the name I always use :3